websites/I/IGDB/presence.ts

```diff
@@ -0,0 +1,103 @@
+const presence = new Presence({
+    clientId: "924944781703020554"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingTimestamp
+    },
+    path = document.location.pathname;
+  if (path === "/discover") presenceData.details = "Discovering games";
+  else if (path.startsWith("/games")) {
+    if (path.indexOf("coming_soon") > -1)
+      presenceData.details = "Viewing future games";
+    else if (path.indexOf("recently_released") > -1)
+      presenceData.details = "Viewing recently released games";
+    else if (path.indexOf("new") > -1)
+      presenceData.details = "Adding a new game";
+    else {
+      delete presenceData.largeImageKey;
+      presenceData.details = `Viewing: ${document.title}`;
+      presenceData.state = document.querySelector<HTMLHeadingElement>(
+        "#content-page > div.loaded > div > div.gamepage-header-info > div.gamepage-summary > div.gamepage-title-container > div > a > h3"
+      ).textContent;
+      presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+        "#content-page > div.loaded > div > div.gamepage-header-info > div.gamepage-cover > img"
+      ).src;
+      presenceData.smallImageKey = "logo";
+      presenceData.smallImageText = "IGDB";
+    }
+  } else if (path.startsWith("/genres")) {
+    if (path === "/genres") presenceData.details = "Browsing all genres";
+    else {
+      (presenceData.details = "Browsing a genre"),
+        (presenceData.state = document.title.slice(0, -5));
```

Comment:
Brackets aren't needed?
```suggestion
      presenceData.details = "Browsing a genre",
      presenceData.state = document.title.slice(0, -5);
```